doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2841,6 +2850,50 @@ should delete the old JVM option and the wrapped password alias, then recreate
 as shown for :ref:`dataverse.pid.datacite.password` but with the EZID alias
 name.
 
+.. _dataverse.pid.crossref.url:
+
+dataverse.pid.crossref.url
+++++++++++++++++++++++++++
+
+CrossRef url used to post metadata.
+
+.. _dataverse.pid.crossref.rest-api-url:
+
+dataverse.pid.crossref.rest-api-url
++++++++++++++++++++++++++++++++++++
+
+CrossRef API url to retrieve metadata information
+
+.. _dataverse.pid.crossref.username:
+
+dataverse.pid.crossref.username
++++++++++++++++++++++++++++++++
+
+CrossRef uses `HTTP Basic authentication <https://en.wikipedia.org/wiki/Basic_access_authentication>`_
+for their APIs.
+- Used in conjunction with :ref:`dataverse.pid.crossref.url` and :ref:`dataverse.pid.crossref.password`.
+
+.. _dataverse.pid.crossref.password:
+
+dataverse.pid.crossref.password
```

Comment:
When you edit this to move it to the multiple provide section, please add a note like the one on DataCite suggesting that more secure options should be used (versus having the password as a plain text jvm-option)

Reply:
This was directly copied from the original PR that was approved. I'll make the change, I'm just wondering why it was approved before

Reply:
The PR was made before multiPid was supported/before docs were improved with that, etc.

Reply:
Will modify the doc